app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/PersonalIdentifier.cs

```diff
@@ -18,7 +18,7 @@ public class PersonalIdentifier : PersonalInfo
         public string IssuedBy { get; set; }
 
         [Description("The owner of the ID e.g. Applicant, Person Sought, e.t.c")]
-        public OwnerType Owner { get; set; }
+        public OwnerType Owner { get; set; } = OwnerType.NotApplicable;
 
```

Comment:
We shouldn't put this in the contract. It should be in the mapping or separate definition